server/example.env

```diff
@@ -32,3 +32,10 @@ AUTH0_AUDIENCE=https://xxxxxxxxxx/api
 # The Domain as a secure URL found in your Auth0 Application settings
 # See https://auth0.com/docs/quickstart/webapp/express/01-login 
 AUTH0_ISSUER_BASE_URL=https://xxxxxxxxx.us.auth0.com/
+
+# CircleCI 
+AUTH0_URL=https://xxxxxxxxx.us.auth0.com/oauth/token
+AUTH0_USERNAME=xxxxxxxxxxxxxx
+AUTH0_PASSWORD=xxxxxxxxxxxxxx
+AUTH0_CLIENT_ID=xxxxxxxxxxxxxx
+AUTH0_CLIENT_SECRET=xxxxxxxxxxxxxx
```

Comment:
Do we use these anywhere else other than CircleCI?

Reply:
No we do not need anywhere else other than CircleCi. I wasn't quite sure if we should add it here or not. But if we don't add it here how will we remember we need these in the CircleCi env?

Reply:
It's ok to put them here if they are required for the server. I was just curious :)

Reply:
Sorry, does that mean they're only used for testing? If so, they should go into some 1password document or the readme since it's only relevant for us (and I suppose anyone who forks and wants to e2e test as well).

Reply:
I believe this is only for documentation. They will go to 1password no matter what.

Reply:
Yes only for testing. I added to `example.env` for documentation, and also added to 1password